src/wallet/rpcwallet.cpp

```diff
@@ -1849,9 +1849,77 @@ UniValue backupwallet(const UniValue& params, bool fHelp)
 
     bool bWalletBackupResults = BackupWallet(*pwalletMain, GetBackupFilename("wallet.dat"));
     bool bConfigBackupResults = BackupConfigFile(GetBackupFilename("gridcoinresearch.conf"));
+
+    std::vector<std::string> backup_file_type;
+
+    backup_file_type.push_back("wallet.dat");
+    backup_file_type.push_back("gridcoinresearch.conf");
+
+    std::vector<std::string> files_removed;
+    UniValue u_files_removed(UniValue::VARR);
+
+    bool bManageBackupResults = ManageBackups(GetBackupPath(), backup_file_type, 0, 0, files_removed);
+
+    for (const auto& iter : files_removed)
+    {
+        u_files_removed.push_back(iter);
+    }
+
     UniValue ret(UniValue::VOBJ);
     ret.pushKV("Backup wallet success", bWalletBackupResults);
     ret.pushKV("Backup config success", bConfigBackupResults);
+    ret.pushKV("Manage backup file retention success", bManageBackupResults);
+    ret.pushKV("number_of_files_removed", (int64_t) files_removed.size());
+    ret.pushKV("files_removed", u_files_removed);
+
+    return ret;
+}
+
+UniValue managebackups(const UniValue& params, bool fHelp)
+{
+    if (fHelp || (params.size() != 0 && params.size() != 2)
+            || (params.size() == 2 && (params[0].get_int() < 0 || params[1].get_int() < 0)))
+        throw runtime_error(
+                "managebackups ( \"retention_by_number\" \"retention_by_days\" )\n"
+                "\nArguments:\n"
+                "1. \"retention_by_number\" (non-negative integer, optional) The number of files to retain\n"
+                "2. \"retention_by_days\"   (non-negative integer, optional) The number of days to retain\n"
+                "These must be specified as a pair if provided."
```

Comment:
Might be helpful to add the requirements to use the command that you mentioned 
```suggestion
                "These must be specified as a pair if provided.\n"
                "To run this command, -managebackupretention must be set as an argument or\n"
                "given in the config file with managebackupretention=1"
```

Reply:
This is a good idea.